README.rst

```diff
@@ -57,10 +56,7 @@ Or on Windows, use the following instead from a CMD prompt::
     > make env
     > .\env\bin\activate
 
-This makefile target is included for convenience ... you may of course choose to
-manage your own virtual environment.  ``requirements.txt``,
-``requirements-dev.txt`` and ``requirements-docs.txt`` list the python
-dependencies needed.
+This makefile target is included for convenience. It uses ``conda`` and installs packages from ``conda-forge``.
 
 Using a different environment name
 """"""""""""""""""""""""""""""""""
```

Comment:
Yes, I think you're right. Since the `env` target of the Makefile calls conda like this:

`(CONDA) create -p $(ENV)`

It's using the `-p` flag, so `ENV` will always be interpreted as a path to a directory. 

>Maybe this is a me problem but the only way I could get a named environment was to specify ENV=path/to/miniconda/base /myEnv

That makes sense; not a you problem. I guess `make env` was designed with the opinion that prefixed environments are a good idea. But creating our own environments in whatever way we prefer as individuals is also a good idea!